app/grandchallenge/components/form_fields.py

```diff
@@ -176,8 +182,13 @@ def get_file_field(self):
                     f"Unknown type for initial value: {self.initial}"
                 )
 
+        extra_help = _join_with_br(
+            f"{file_upload_text} {self.instance.file_extension}",
+            self.example_download_link,
+        )
+
         self.kwargs["widget"] = FlexibleFileWidget(
-            help_text=self.help_text,
+            help_text=_join_with_br(self.help_text, extra_help),
```

Comment:
After the new implementation of file widget #8ae9479 
The download link is not showing correctly.
Looking into it.

Reply:
Thomas is still in the process of heavily refactoring the logic there, so maybe best to wait until he is done.

Reply:
Got it, I will wait then.